utilization.md

```diff
@@ -379,55 +349,36 @@ A `HostMirror` view is allocated in the host memory space as a mirror of a devic
 - [Kokkos Tutorials - Exercise 3](https://github.com/kokkos/kokkos-tutorials/blob/main/Exercises/03/Solution/exercise_3_solution.cpp)
 <!--#endif-->
 
-#### Create and allocate
+#### Always create and allocate
```

Comment:
Not clear what `create` means and how `always create` is different.

Reply:
Maybe "Create and always allocate" instead? This is to make a difference with the different versions, where the allocation is conditional.

Reply:
Looks clearer.